src/components/choreographer/CdrChoreographer.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
```

Comment:
I didn't realize we were including this in v15?

Reply:
Not exported. Leaving it in for further experimentation

Reply:
I think the idea is we are not speaking about it